app/src/main/res/xml/appfilter.xml

```diff
@@ -23803,6 +23803,7 @@
 	<item component="ComponentInfo{io.github.sds100.keymapper/io.github.sds100.keymapper.ToastActivity}" drawable="keymapper"/>
 	<item component="ComponentInfo{io.github.sds100.keymapper/io.github.sds100.keymapper.mappings.assistant.DeviceAssistantActivity}" drawable="keymapper"/>
 	<item component="ComponentInfo{io.github.sds100.keymapper/io.github.sds100.keymapper.SplashActivity}" drawable="keymapper"/>
+	<item component="ComponentInfo{io.github.sds100.keymapper/io.github.sds100.keymapper.MainActivity}" drawable="key_mapper"/>
```

Comment:
Please verify that the drawable name at the end matches the icon's name as well, like the one above :)
If you're unsure about the name, you can use https://arcticons.com/search to search the icon!